mi/dataset/parser/test/test_zplsc_c.py

```diff
@@ -21,7 +21,8 @@
 
 MODULE_NAME = 'mi.dataset.parser.zplsc_c'
 CLASS_NAME = 'ZplscCRecoveredDataParticle'
-PARTICLE_TYPE = 'zplsc_c_recovered'
+# PARTICLE_TYPE = 'zplsc_c_recovered'
```

Comment:
Are we keeping this commented line for reference or can it be removed?

Reply:
please remove

Reply:
Removed the comment.